engines/prover.cpp

```diff
@@ -148,8 +149,7 @@ void Prover::print_coi_info()
     cout << "  " << statevar << "\n";
 
   cout << "constraints: \n";
-  for (auto constr : ts_.constraints())
-    cout << "  " << constr << "\n";  
+  for (auto constr : ts_.constraints()) cout << "  " << constr << "\n";
```

Comment:
is this formatted by git-clang-format?

Reply:
Yeah it must have been because I didn't intend to change that code otherwise.